plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotMetadata.java

```diff
@@ -494,10 +494,6 @@ private static PinotColumnHandle resolveAggregateExpressionWithAlias(PinotColumn
 
     private static <K, V> V getFromCache(LoadingCache<K, V> cache, K key)
     {
-        V value = cache.getIfPresent(key);
```

Comment:
IIRC there was some mention that `getIfPresent` is more contention safe than `get`? Or was it about `computeIfAbsent`?

I'll check and get back.

Reply:
However this doesn't seem to be in a hot path so shouldn't matter. Readability first.

Reply:
@hashhar thanks for reminding me.
yes, i think it doesn't matter here
i think this is the thread: https://github.com/trinodb/trino/pull/9980

cc @sopel39